File: recorrido_matriz_2d (1).py
"""
Tarea: Recorrido de matriz 2D (Python)
Ejecutar con: python recorrido_matriz_2d.py
"""

matriz = [
    [8, 3, 5],
    [1, 9, 2],
    [7, 4, 6]
]

print("Matriz 2D:")
for fila in matriz:
    print("\t".join(str(valor) for valor in fila))

File: RecorridoMatriz2D (1).java
/*
 * Tarea: Recorrido de matriz 2D (Java)
 */
public class RecorridoMatriz2D {
    public static void main(String[] args) {
        int[][] matriz = {
            {8, 3, 5},
            {1, 9, 2},
            {7, 4, 6}
        };

        System.out.println("Matriz 2D:");
        for (int fila = 0; fila < matriz.length; fila++) {
            for (int col = 0; col < matriz[fila].length; col++) {
                System.out.print(matriz[fila][col] + "\t");
            }
            System.out.println();
        }
    }
}
